CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ set (DIR_INCLUDE ${DIR_PREFIX}/include/)
 set (DIR_SHARE ${DIR_PREFIX}/share/)
 set (DIR_ETC ${DIR_PREFIX}/etc/)
 set (LIB_SUFFIX "")
-set (DIR_LIBRARY ${DIR_PREFIX}/lib${LIB_SUFFIX}/)
+set (DIR_LIBRARY ${LIB_INSTALL_DIR})
```

Comment:
What is the problem with `CMAKE_INSTALL_PREFIX` on line 67? Can `LIB_SUFFIX` help?

/cc @pprkut